fab/operations/supervisor.py

```diff
@@ -277,10 +278,32 @@ def restart_all_except_webworkers():
     _services_restart()
 
 
+@roles(ROLES_STATIC)
+def _decommission_host(host):
+    files.comment(
```

Comment:
we don't want to remove the entire site, just the one host in the upstream config